lib/rules/object-curly-newline.js

```diff
@@ -180,7 +180,7 @@ module.exports = {
                         fix(fixer) {
                             return fixer.removeRange([
                                 openBrace.range[1],
-                                first.range[0]
+                                firstTokenOrComment.range[0]
```

Comment:
Hm I see. I agree that in that case it should do nothing, because fixing the new line issue here would require changing the comment from `// comment` to `/* comment */`
```js
// Closing brace is accidentally commented out.
var foo = {// comment};
// Comment syntax changed.
var foo = {/* comment */};
```
I'll address soon and update the PR.

Now I am wondering if my change handles correctly the following case
```js
var foo = {
  // comment
};
```
The tests I added don't use this style of comments, will update the tests cases and fix the code if necessary. Thanks.